client/my-sites/checkout/checkout-thank-you/remove-after-traslated/new-congrats-translations.jsx

```diff
@@ -0,0 +1,196 @@
+
+	// Paid single plugin
+	translate( 'Your site, more powerful than ever' ),
+	translate( 'Use the plugin' ),
+	translate( 'Manage plugin' ),
+	translate( 'Need help setting your plugin up?' ),
+	translate(
+		'Check out our support documentation for step-by-step instructions and expert guidance on your plugin set up.'
+	),
+	translate( '{{a}}Plugin setup guide{{/a}}', {
+		components: {
+			a: <a href="https://wordpress.com/support/plugins/use-your-plugins/" />,
+		},
+	} ),
+	translate( 'All-in-one plugin documentation' ),
+	translate( `Unlock your plugin's potential with our comprehensive support documentation.` ),
+	translate( 'Plugin documentation' ),
+
+	// Paid multiple plugin
+	translate(
+		'All set! Time to put your new plugin to work and take your site further.',
+		'All set! Time to put your new plugins to work and take your site further.',
+		{
+			count: 2,
+		}
+	),
+	translate( 'Use the plugins' ),
+
+	// Plan / Annual / Without custom domain selected or included in the cart
+	translate( 'Get the best out of your site' ),
+	translate( 'All set! Start exploring the features included with your %(plan)s plan', {
+		args: {
+			plan: translate( 'Free' ),
+		},
+	} ),
+	translate( '%(plan)s plan', {
+		args: {
+			plan: translate( 'Free' ),
+		},
+		comment: `Shows which plan the user is on. For example, "Free plan"`,
+	} ),
+	translate( `Let’s work on the site` ),
+	translate( 'Manage plan' ),
+	translate( 'Claim your free domain ' ),
+	translate(
+		'Easy to remember. Easy to share. And free for your first year. Help people find you with a personalized site address.'
+	),
+	translate( '{{a}}Claim custom domain{{/a}}', {
+		components: {
+			a: <a href="https://wordpress.com/domains/add/" />,
+		},
+	} ),
+	translate( 'Everything you need to know' ),
+	translate( 'Explore our support guides and find an answer to every question.' ),
+	translate( '{{a}}Explore support resources{{/a}}', {
+		components: {
+			a: <a href="https://wordpress.com/support/" />,
+		},
+	} ),
+
+	// Plan monthly or annual with custom domain for Business plans and above
+	translate( 'There’s a plugin for that' ),
+	translate(
+		'With 54,000+ plugins and apps, you’ll never outgrow your website. If you can think of it, there’s a plugin to make it happen.'
+	),
+	translate( '{{a}}Discover plugins{{/a}}', {
+		components: {
+			a: <a href="https://wordpress.com/plugins/" />,
+		},
+	} ),
+
+	// Plan monthly or annual with custom domain for plans below Business
+	translate( 'A site refresh' ),
+	translate(
+		'A new look and feel can help you stand our from the crowd. Get a new theme and make an impression.'
+	),
+	translate( '{{a}}Find your new theme{{/a}}', {
+		components: {
+			a: <a href="https://wordpress.com/themes/" />,
+		},
+	} ),
+
+	// Domains
+	translate( 'Your own corner of the web' ),
+	translate(
+		'All set! We’re just setting up your new domain so you can start spreading the word.'
+	),
+	translate( 'Share site' ),
+	translate( 'Site copied' ),
```

Comment:
Another update on Figma (VLoPHWqcewxNKZYjjkDu3O-fi-2446_3296)
```suggestion
	translate( 'Manage domain' ),
```

Reply:
We already have `Manage domain` in the codebase so that has been translated.
Site copied came from [here](https://docs.google.com/spreadsheets/d/1LW1mCbubP34-xYQLFDRlG85dwsQk7iI7WvaOmcpSqpI/edit#gid=0): Share site (copy domain) and change CTA to Site copied

Reply:
Thanks @candy02058912